Makefile

```diff
@@ -26,7 +35,7 @@ install:
 	cp runc /usr/local/bin/runc
 
 clean:
-	rm runc
+	rm runc || true
```

Comment:
I think the intention of the author is that " if the file runc is non-existed, it shouldn't fail".

if so, I prefer to
```sh
test ! -e runc || rm runc
```